boards/arm/stm32f0_disco/doc/stm32f0_disco.rst

```diff
@@ -0,0 +1,141 @@
+.. _stm32f0_disco_board:
+
+ST STM32F0DISCOVERY
+################
+
+Overview
+********
+
+The STM32F0 Discovery development board with STM32F051R8T6 MCU
+
+The STM32 Discoery board integrates the ST-LINK/V2-1 debugger and programmer.
+
+The STM32 Discovery board comes with the STM32 comprehensive software HAL library together
+with various packaged software examples.
+
+.. image:: img/en.stm32f0discovery.jpg
+     :width: 350px
+     :height: 404px
+     :align: center
+     :alt: STM32F0DISCOVERY
+
+More information about the board can be found at the `STM32F0DISCOVERY website`_.
+
+Hardware
+********
+
+- STM32F051R8T6 microcontroller featuring 64 KB Flash memory, 8 KB RAM in an LQFP64 package
+- On-board ST-LINK/V2 with selection mode switch to use the kit as a standalone ST-LINK/V2 (with SWD connector for programming and debugging)
```

Comment:
try to keep line length < 80 characters (unless you've got a long URL):

    - On-board ST-LINK/V2 with selection mode switch to use the kit as a standalone 
      ST-LINK/V2 (with SWD connector for programming and debugging)